ldTest run scala.scalanative.junit.MultiAssumeFail1Test started
ldTest scala.scalanative.junit.MultiAssumeFail1Test.multiTest1 started
lwTest assumption in test scala.scalanative.junit.MultiAssumeFail1Test.multiTest1 failed: org.junit.AssumptionViolatedException: This assume should not pass, took <TIME>
e3scala.scalanative.junit.MultiAssumeFail1Test.multiTest1::org.junit.AssumptionViolatedException: This assume should not pass::true
ldTest scala.scalanative.junit.MultiAssumeFail1Test.multiTest1 finished, took <TIME>
e0scala.scalanative.junit.MultiAssumeFail1Test.multiTest1::::true
ldTest scala.scalanative.junit.MultiAssumeFail1Test.multiTest2 started
ldTest scala.scalanative.junit.MultiAssumeFail1Test.multiTest2 finished, took <TIME>
e0scala.scalanative.junit.MultiAssumeFail1Test.multiTest2::::true
ldTest scala.scalanative.junit.MultiAssumeFail1Test.multiTest3 started
ldTest scala.scalanative.junit.MultiAssumeFail1Test.multiTest3 finished, took <TIME>
e0scala.scalanative.junit.MultiAssumeFail1Test.multiTest3::::true
ldTest scala.scalanative.junit.MultiAssumeFail1Test.multiTest4 started
ldTest scala.scalanative.junit.MultiAssumeFail1Test.multiTest4 finished, took <TIME>
e0scala.scalanative.junit.MultiAssumeFail1Test.multiTest4::::true
ldTest scala.scalanative.junit.MultiAssumeFail1Test.multiTest5 started
ldTest scala.scalanative.junit.MultiAssumeFail1Test.multiTest5 finished, took <TIME>
e0scala.scalanative.junit.MultiAssumeFail1Test.multiTest5::::true
ldTest run scala.scalanative.junit.MultiAssumeFail1Test finished: 0 failed, 0 ignored, 5 total, <TIME>
d
